COMP 120 Tinkering Graphics-JH182233/Program/Option.py

```diff
@@ -0,0 +1,205 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures")
+global picture1 
+picture1 = makePicture("Tree-Four-Seasons.jpg")
+global picture2 
+picture2 = makePicture("Tree-Winter.jpg")
+global picture3 
+picture3 = makePicture("Tree-Spring.jpg")
+global picture4 
+picture4 = makePicture("Tree-Summer.jpg")
+global picture5 
+picture5 = makePicture("Tree-Autumn.jpg")  
+global picture6
+picture6 = makePicture("tree.jpg")
+global picture7
+picture7 = makePicture("test.jpg")
+global picM
+picM = makeEmptyPicture(513, 556, black)
+
```

Comment:
It may be better to store the pictures in an array rather than creating separate variables for them.